ui/src/timeMachine/reducers/index.ts

```diff
@@ -281,7 +281,7 @@ export const timeMachineReducer = (
         draftState.queryResults.status = status
         draftState.queryResults.errorMessage = errorMessage
 
-        if (files) {
+        if (files && files.length) {
```

Comment:
[`files`](https://github.com/influxdata/influxdb/blob/master/ui/src/timeMachine/actions/queries.ts#L47) is typed as:

```ts
files?: string[]
```
but iin these flows, it's passed in as `[]`